plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ui/ServerFlagSelectionDialog.java

```diff
@@ -78,28 +78,32 @@ protected Control createDialogArea(Composite parent) {
 
       @Override
       public IStatus validate(Object[] selection) {
-        if (selection.length == 1) {
+        if (selection == null || selection.length == 0) {
+          return new Status(OK, Activator.PLUGIN_ID, "");
```

Comment:
I thought about it, but that would use plugin ID as null. Is that OK?

On Thu, Apr 21, 2016, 21:45 Brian de Alwis <notifications@github.com> wrote:

> In
> plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ui/ServerFlagSelectionDialog.java
> <https://github.com/GoogleCloudPlatform/gcloud-eclipse-tools/pull/93#discussion_r60681431>
> :
>
> > @@ -78,28 +78,32 @@ protected Control createDialogArea(Composite parent) {
> >
> >        @Override
> >        public IStatus validate(Object[] selection) {
> > -        if (selection.length == 1) {
> > +        if (selection == null || selection.length == 0) {
> > +          return new Status(OK, Activator.PLUGIN_ID, "");
>
> return Status.OK_STATUS;
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-eclipse-tools/pull/93/files/4cd854b7d4619f4f0776c1ffda36bb42188e34df..e11ae96e16bb3b67c8b2152f2663cdc81483b5ed#r60681431>
>